framework/web/ContentRangeHttpException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\web;
+
+/**
+ * ContentRangeHttpException represents an exception caused by an improper request of the end-user.
+ * This exception uses when the requested range is not satisfiable.
+ *
+ * Throwing an ContentRangeHttpException like in the following example will result in the 416 page to be displayed.
+ *
+ * @author Zalatov Alexander <CaHbKa.Z@gmail.com>
+ *
+ * @since 2.0.11
+ */
+class ContentRangeHttpException extends HttpException
```

Comment:
Exception name is inconsistent - all other HttpExceptions uses [status code name](https://github.com/yiisoft/yii2/blob/master/framework/web/Response.php#L163) as exception name, so to this should be named `RangeNotSatisfiableHttpException`.